lib/hashes.md

```diff
@@ -5,40 +5,46 @@ instructor_notes: Feel free to re-organize the headings (or add/remove headings)
 
 # What is a Hash?
 
-Replace me with your response...
+A hash is a collection of objects, like an array, but they dont have to be numbers.  Hashes contain a set of 'keys' and 'values', where
+the values dont need to be numbers.
 
 # What are some examples of information that would be Hashes as opposed to some other data type?
 
-Replace me with your response...
+The information used in a hash would be lists of items, to be set for a given value.  A ranking of different teams and their given value in different
+areas, such as strength, speed, debth, etc, or a list of ingredients on a pizza.
 
 # How are Hashes and Arrays similar? How are they different?
 
-Replace me with your response...
+Hashes and arrays are both lists, but hashes dont have to use numbers to index them, or describe them, they can be terms or definitions.
 
 # How do you retrieve a particular value from a Hash?
 
-Replace me with your response...
+A value from a hash can be retrieved by 'put'ing or 'print'ing the value like a variable.
 
 # How do you add information to a Hash?
 
-Replace me with your response...
+A hash is built by defineing the hash, and puting given keys to be equal to or greater than the given value.
```

Comment:
Gotcha. It's a bit confusing to refer to that symbol with "equal to or greater than", because it makes it sound like there is _math_ being done. We usually refer to that symbol as a "rocket".